.github/workflows/ios-unsigned.yml

```
@@ -64,65 +74,365 @@ jobs:
           working-directory: ios
 
+      - name: Ensure CI helper scripts (collect symbols)
+        shell: bash
+        run: |
+          set -euo pipefail
+          mkdir -p scripts/ci
+          node -e "
+            const fs = require('fs');
+            fs.mkdirSync('scripts/ci', { recursive: true });
+            const s = `#!/usr/bin/env bash
+set -euo pipefail
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 86: could not find expected ':'

(syntax-check)